### DISCUSSION TITLE
Hunt groupby IP no results

### DISCUSSION THREAD

**Discussion by shawnpersona** [Security-Onion-Solutions/securityonion#8903] [Unsupported Versions]
Greetings,

I seem to be having an issue with getting a groupby of Source IP, Destination IP and port to bring back any results in Hunt.

When I use `event.dataset:conn | groupby source.ip destination.ip network.protocol destination.port` I am getting
timeout of 300000ms exceeded

On a side note when I try to use `Group By Sensor, Source IP/Port, Destination IP/Port, Name` in Alerts, I do not get any results

![2022-10-11 14_12_26-! Security Onion - Alerts - Group By Name, Module](https://user-images.githubusercontent.com/98039048/195189289-e9d598cf-6132-4220-ae9e-4440ab1e1638.png)

This happens regardless if I set 1 hour or 1 week.

Running version 2.3.160
I am running a ManagerSearch node have about 24 TB of HDD space, 256 GB memory and 2 Xeon 12 core CPU
Currently have 8 sensor nodes from other sites Each with Large drives, and 16 Gb memory,  Xeon CPU
1 IDH node
I am ingesting eventlogs including sysmon using winfilebeat from multiple servers
I am also bringing in syslog data from my firewalls.

I am hoping someone can help me with troubleshooting this.

**Comment by dougburks** [comment]
How much data do you have in Elasticsearch?

Do you just have a ManagerSearch node or do you have also have separate search nodes?

Are you using rotational drives or SSD?

**Reply by shawnpersona** [reply]
Hi,

I am really not sure how to tell how much data I have in elastic.  

Currently I only have a ManagerSearch and no search nodes.

Drives are SAS non SSD

**Reply by dougburks** [reply]
> I am really not sure how to tell how much data I have in elastic.

You can take a look at the following:
```
du --max-depth=1 -h /nsm/elasticsearch/
```
```
sudo so-elasticsearch-indices-list
```

You may need to tune Elasticsearch, add more search nodes, or move to SSD:
https://docs.securityonion.net/en/2.3/elasticsearch.html
https://docs.securityonion.net/en/2.3/architecture.html#distributed
https://docs.securityonion.net/en/2.3/hardware.html#elastic-stack
